packagingbuild/Dockerfile.template

```diff
@@ -122,7 +122,7 @@ RUN echo -e "#!/bin/sh\nexit 101\n" > /usr/sbin/policy-rc.d && \
 {% if (version == '8') -%}
 # Install development tools and python3 for EL8
  RUN yum -y install python3 python3-devel rpmdevtools python3-virtualenv && \
-     pip3 install virtualenv==16.6.0 pip==19.1.1 wheel setuptools
+     pip3 install virtualenv==20.2.2 pip==20.3.3 wheel setuptools
```

Comment:
Need to also amend the line further down that still pins virtualenv to 16.6.0 for el7

Reply:
Done, thanks, did not catch that one.